### ISSUE TITLE
URL to RSS feed not marked-up proberly in HTML

### ISSUE THREAD

**Issue by agharbeia** [hatnote/top#32]
The RSS feed is only reachable via the link at the bottom of the page.
Ideally, the HTML page should properly advertise the URL of the feed via a <LINK> element in its markup to facilitate client discovery (the little button in the browser which lights up indicating the availability of a feeds of this page)

**Comment by mahmoud** [comment]
Good point! @slaporte if you get to this before me, here's an example i'm pretty sure works: https://github.com/mahmoud/chert/blob/master/chert/scaffold/themes/sedimental/base.html#L8

More info here: https://blog.superfeedr.com/rss-autodiscovery/

Thanks for the report, @agharbeia 

PS Stephen: re: feed validation, I was going to fix that up at some point, but also just in case you wanted to give it a shot, I'm pretty sure chert's validates fine: https://github.com/mahmoud/chert/blob/master/chert/atom.xml
